UPGRADE.md

```diff
@@ -83,7 +83,7 @@ it is using.
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
 >
-    <key>categories</key>
+    <key>category_details</key>
```

Comment:
Line 101 missing formKey should also be changed.

Also if we use details the route name should also be details not only detail: sulu_category.edit_form.detail **`s`**

```
return [
    (new Route('sulu_category.edit_form.details', '/details', 'sulu_admin.form'))
	        ->setOption('tabTitle', 'sulu_category.details')
	        ->setOption('resourceKey', 'categories')
	        ->setOption('formKey', 'category_details')
	        ->setOption('backRoute', 'sulu_category.datagrid')
];
```